storage/migrations/8_heaviest_chain.go

```diff
@@ -0,0 +1,42 @@
+package migrations
+
+import (
+	"github.com/go-pg/migrations/v8"
+)
+
+// Schema version 8 adds a heaviest chain view
+
+func init() {
+	up := batch(`
+CREATE MATERIALIZED VIEW IF NOT EXISTS heaviest_chain_view AS
+WITH RECURSIVE heaviest_chain AS (
+	SELECT
+		b.cid,
+		b.height,
+		b.miner,
+		b.timestamp,
+		b.parent_state_root,
+		b.win_count
```

Comment:
Why the `b.win_count` field?

Reply:
I'm not sure, @placer14 wrote this originally

Reply:
Because we want all of our data in our derived view without needing to join on write-heavy DBs.